### ISSUE TITLE
install rules for libraries are incorrect

### ISSUE THREAD

**Issue by gavanderhoorn** [ros-industrial/industrial_core#78]
According to the _Installing_ section of the [Building and installing C++ libraries and headers](http://docs.ros.org/hydro/api/catkin/html/howto/building_libraries.html) page of the catkin documentation, library targets should be installed using:

```
install(TARGETS your_library
        ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
        LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
        RUNTIME DESTINATION ${CATKIN_GLOBAL_BIN_DESTINATION})
```

At least `industrial_robot_client` and `simple_message` do not do this. This should be corrected.

**Comment by mathias-luedtke** [comment]
(from `industrial_robot_client`)
```
install(
    TARGETS industrial_robot_client industrial_robot_client_bswap
    LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION})
```
is correct as well, but It looks like the install targets and the exports needs an overhaul in general.

**Comment by ipa-nhg** [comment]
solved by: https://github.com/ros-industrial/industrial_core/pull/193

The previous version was also ok (I checked the install workspace), but make sense follow the official documentation.

**Comment by gavanderhoorn** [comment]
Fixed by #193.
